### PR TITLE
Use an enum & helper method for setting the active admin nav link

### PR DESCRIPTION
**Pull request by gunndabad**
Fixes the QA link not being active after splitting the controller.

### CODE REVIEW

File: src/Dfc.CourseDirectory.WebV2/SharedViews/Components/AdminTopNav.cshtml [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
-    var currentController = (string)ViewContext.RouteData.Values["Controller"];
-    var currentAction = (string)ViewContext.RouteData.Values["Action"];
+    var activeSection = ViewData["ActiveAdminTopNavSection"] as AdminTopNavSection?;
 }
 
 <button type="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
 <nav>
     <ul id="navigation" class="govuk-header__navigation" aria-label="Top Level Navigation">
-        <li class="govuk-header__navigation-item@(currentController == "HelpdeskDashboard" && currentAction == "Dashboard" ? " govuk-header__navigation-item--active" : null)">
+        <li class="govuk-header__navigation-item@(activeSection == AdminTopNavSection.Home ? " govuk-header__navigation-item--active" : null)">
```
Comment (jameslawesynergy) [inline src/Dfc.CourseDirectory.WebV2/SharedViews/Components/AdminTopNav.cshtml:10]: I'm sure I've seen this done before with a `TagHelper` of some sort...?

**Review by jameslawesynergy — COMMENTED** [review]
I'm not sure I'm a fan of this approach. Feels like there should be a better way to highlight some links, right?

File: src/Dfc.CourseDirectory.WebV2/SharedViews/Components/AdminTopNav.cshtml (hunk 1 above)
Comment (gunndabad) [reply src/Dfc.CourseDirectory.WebV2/SharedViews/Components/AdminTopNav.cshtml:10]: You mean the active section detection?

Comment (jameslawesynergy) [reply src/Dfc.CourseDirectory.WebV2/SharedViews/Components/AdminTopNav.cshtml:10]: Possibly. It's been a while and I've not looked into it. Let's discuss tomorrow.

**Review by jameslawesynergy — APPROVED** [review]

**Pull request merged**